vertx-web/src/main/java/io/vertx/ext/web/handler/impl/StaticHandlerImpl.java

```diff
@@ -690,5 +774,13 @@ boolean isOutOfDate() {
 
   }
 
-
+  public String getLinkExtensionAs(String fileName) {
```

Comment:
Is this the right way to add the 'as' attribute for Link header?
Any suggestions are highly appreciated.

Reply:
This is not correct i believe. https://www.w3.org/TR/preload/#as-attribute

the `as` is not a mime type but an extension target attribute see: https://tools.ietf.org/html/rfc5988

Reply:
Yes, but I'm not sure where I can specify the mapping for the file to extension target attribute unless explicitly provided by the user.
If the setHttp2Mapping() method itself takes this extension target attribute as param, then I guess this method can be removed.